src/test/java/pl/widulinski/scrapp/createExcel/CreateExcelTests.java

```diff
@@ -0,0 +1,91 @@
+package pl.widulinski.scrapp.createExcel;
+
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import pl.widulinski.scrapp.FoundWebElements.FoundWebElement;
+import pl.widulinski.scrapp.testHelper.ExcelReader;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Stream;
+
+
+@ExtendWith(SpringExtension.class)
+@Slf4j
+@SpringBootTest
+public class CreateExcelTests {
+
+
+    @Test
+    public void testCreateExcel() throws IOException {
+        //given
+        CreateExcel createExcel = new CreateExcel();
+        String fileTestPath = "C:\\Users\\Dawid\\IdeaProjects\\ScrApp\\";
```

Comment:
lokalna ściezka - wczytaj z resourców tym bardziej że spring robi to w dosłownie jednej linijce np. `@CsvSource` ale są inne opcje